app/views/products/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<% provide :title, t("product.name") %>
+<div class="row">
+  <div class="col-12 col-md-9">
+    <div class="row">
+      <div class="col-12 col-md-6">
+          <img class="d-block w-100" src="https://www.mageplaza.com/assets/img/extensions/product-labels.png">
+      </div>
+      <div class="col-12 col-md-6 pl-3">
+        <h3 class="mt-4"><%= @product.name %></h3>
+        <small>Số lượng còn lại: <span id="quantity"><%= @product.quantity %></span></small>
+        <p>Giá: <%= number_to_currency @product.price %></p>
+        <%= form_for :cart, url: add_to_cart_path, remote: true do |f| %>
+          <%= hidden_field_tag :product_id, @product.id %>
+          <div class="form-group">
+            <label class="mr-3 ml-3">Số lượng: </label>
+            <%= f.number_field :quantity, value: 1, in: 1..5 %>
```

Comment:
`, in: 1..5` sao phải rang buộc là max 5 thôi chú,  họ mua nhìu quá ko cho ah?
Nên để là max lấy từ product chứ. ai ki bo vậy lần bán có 5 cái

Reply:
Tại vì sau khi đặt hàng e sẽ trừ thẳng vào quantity còn lại. Nên em sợ k giới hạn thì họ dễ spam để web em hết hàng :v